app/controllers/admin/events_controller.rb

```diff
@@ -31,7 +31,7 @@ def event
     end
 
     def events
-      @events ||= ::Event::List.(events: search_against(Event), page: params[:page])
+      @events ||= ::Event::List.(events: search_against(Event), page: params[:page]).per(10)
```

Comment:
Hardcoded values are not good (`10`)
This `10` value could be extracted to settings.yml, but
@sveredyuk wrote cool engine to use settings throughout the app - [ez-settings](https://github.com/ez-engines/ez-settings)
I think we need to use it to be consistent.
@sveredyuk do you agree?